README.md

```diff
@@ -566,15 +566,15 @@ spec:
     volumes: .spec.template.spec.volumes
 ```
 
-Note: This example is equivalent to not specifying a mapping or specifying an empty mapping.
+> Note: This example is equivalent to not specifying a mapping or specifying an empty mapping.
 
 ## Runtime Behavior
 
 When a `ClusterWorkloadResourceMapping` is defined in the cluster matching a workload resource it **MUST** be used to map the binding that type. If no mapping is available for the type, the implementation **MUST** treat the workload resource as a `PodSpec`-able type.
 
 If a `ServiceBinding` specifies `.spec.workload.containers` and a `MappingContainer` specifies a `name` expression, the resolved name **MUST** limit which containers in the workload are bound. If either key is not defined, the container **SHOULD** be bound.
 
-An implementation **MUST** create empty values at locations referenced by [Restricted JSONPaths](#restricted-jsonpath) that do not exist on the workload resource. Values referenced by JSONPaths in both the `MappingTemplate` and `MappingContainer`s **MUST** be mutated by a `ServiceBinding` reconciler as if they were defined directly by a PodTemplateSpec. A reconciler **MUST** preserve fields on the workload resource that fall outside the specific fragments and types defined by the mapping.
+An implementation **MUST** create empty values at locations referenced by [Restricted JSONPaths](#restricted-jsonpath) that do not exist on the workload resource. Values referenced by JSONPaths in both the `MappingTemplate` and `MappingContainer` objects **MUST** be mutated by a `ServiceBinding` reconciler as if they were defined directly by a PodTemplateSpec. A reconciler **MUST** preserve fields on the workload resource that fall outside the specific fragments and types defined by the mapping.
```

Comment:
Do we consider `MappingTemplate` and `MappingContainer` to be objects? They certainly are not k8s objects.

Reply:
In one of the previous paragraphs, it's written like this:
```
A `MappingTemplate` object **MUST** define at least one `version` entry...
```
How about using the word "section" instead of "object" everywhere?

Reply:
I can see the usage of "fragments" in the last sentence which looks good to me. I am going to use the same word in the other sentences. Thanks for the suggestion!

Reply:
Changed to fragments: fa9d402871e9a43b2590fa592857b8fe0d6c1370